test/integration/suite_test.go

```diff
@@ -38,9 +40,22 @@ func TestMain(m *testing.M) {
 
 	fmt.Println("INFO: setting up test environment")
 	kongbuilder := kong.NewBuilder()
+	if enterpriseEnablement == "on" {
+		if enterpriseRepo != "" && enterpriseTag != "" {
+			licenseJSON := os.Getenv("KONG_ENTERPRISE_LICENSE")
+			if licenseJSON == "" {
+				panic("enterprise can not be installed w/o a license json.")
+			}
+			kongbuilder = kongbuilder.WithEnterprise().WithImage(enterpriseRepo, enterpriseTag).WithEnterpriseLicense(licenseJSON).WithKongAdminPassword(KongTestPassword).WithAdminServiceTypeLoadBalancer()
```

Comment:
nit: once a builder style line gets this long, it can be helpful for better readability to put each option on its own line:

```suggestion
			kongbuilder = kongbuilder.WithEnterprise().
			    WithImage(enterpriseRepo, enterpriseTag).
			    WithEnterpriseLicense(licenseJSON).
			    WithKongAdminPassword(KongTestPassword).
			    WithAdminServiceTypeLoadBalancer()
```

Reply:
https://github.com/Kong/kubernetes-ingress-controller/pull/1758/files#diff-b5e061fd98698fd6cfe27cc0e162263d736df3560a8b8d58e80bab161887c7b8R49